source/troubleshooting.txt

```diff
@@ -173,3 +173,144 @@ HOMEDRIVE, HOMEPATH, or USERPROFILE are blank
 
 This error might appear if the {+atlas-cli+} can't access your home
 directory.
+
+
+# Are there any known errors to add for ``atlas deployments``?
+
+.. _local-deploy-errors:
+
+Local Deployment Errors
+-----------------------
+
+Error: qemu exited unexpectedly with exit code -1, stderr: qemu-system-x86_64: Error: HV_DENIED
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+atlas-cli+} uses `Podman </https://podman.io/>`__ and `QEMU 
+<https://www.qemu.org/>`__. We have identified this issue on MacOS x86 
+architectures. Upgrade to the latest version of QEMU and restart Podman.
+
+Error: exit status 127
+~~~~~~~~~~~~~~~~~~~~~~
+
+This error might occur when you use the ``atlas deployments setup`` 
+command on Ubuntu Linux because of a `known issue 
+<https://bugs.launchpad.net/ubuntu/+source/libpod/+bug/2024394>`__ with 
+Podman. Downgrade Podman:
+
+.. code-block:: sh
+
+   sudo apt install podman=3.4.4+ds1-1ubuntu1
+
+MongoNetworkError: connect ECONNREFUSED
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This issue might occur after your machine goes into sleep mode or 
+restarts. When you try to connect to the local {+cluster+}, this error 
+occurs.
+
+The {+atlas-cli+} uses `Podman </https://podman.io/>`__ and `QEMU 
+<https://www.qemu.org/>`__ to run dockerized instances of MongoDB. 
+Podman has a daemonless architecture. So, the local MongoDB instances 
+don't restart automatically.
+
+To fix this issue:
+
```

Comment:
Should we use atlas deployments start commands instead? @andreaangiolillo

Reply:
yes, you can update this section to ask users to run

`atlas deployment start <deploymentName>`